tests/tuning/hyperparams/test_boolean.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+
+from unittest import TestCase
+
+from btb.tuning.hyperparams.boolean import BooleanHyperParam
+
+
+class TestBooleanHyperParam(TestCase):
+    """Unit test for the class ``BooleanHyperParam``."""
+
+    def setUp(self):
+        self.instance = BooleanHyperParam()
+
+    def test_trasnform_list(self):
+        """Test that the method ``transform`` performs a normalization over a ``list`` of boolean
+        values and converts them in to a search space of [0, 1]^k.
+        """
+
+        # setup
+        values_1 = [True, False, True]
+        values_2 = [False, False, True]
+
+        # run
+        result_1 = self.instance.transform(values_1)
+        result_2 = self.instance.transform(values_2)
+
+        # assert
+        assert all(isinstance(res[0], int) for res in result_1.tolist())
```

Comment:
- does iteration require `tolist` be called?
- be on the lookout for when you should use more specific assertion methods from the unittest library, which significantly aids in debugging and in clarifying your intentions to the reader: https://docs.python.org/3/library/unittest.html#assert-methods